low-level/files/opt/seagate/sspl/setup/resource_map/lr2/storage.py

```diff
@@ -65,12 +65,15 @@ def __init__(self):
 
     def validate_storage_type_support(self):
         """Check for supported storage type."""
-        storage_type = Conf.get(
-            GLOBAL_CONF, STORAGE_TYPE_KEY, "virtual").lower()
+        storage_type = Conf.get(GLOBAL_CONF, STORAGE_TYPE_KEY)
```

Comment:
Just FYI, Provisioner supports following storage types:
`RBOD, JBOD, EBOD, virtual`

Reply:
Okay. 
As per design, resource health generation is supported for all the above except virtual storage type.